frontend/src/App.tsx

```diff
@@ -40,10 +44,25 @@ export const App = ({
               path="/transferableCredits"
               component={TransferableCreditScreen}
             />
+            <Route path="/advisor" component={AdvisorRouter} />
             <Route path="/" component={Onboarding} />
           </Switch>
         </Router>
       </PersistGate>
     </Provider>
   );
 };
+
+const AdvisorRouter = (props: any) => {
+  const { path } = props.match;
+
+  return (
+    <GenericAdvisingTemplateComponent>
+      <Switch>
+        <Route path={`${path}/notifications`} component={NotificationsPage} />
+        <Route path={`${path}/manageStudents`} component={ManageStudents} />
+        <Route path={`${path}/templates`} component={TemplatesPage} />
+      </Switch>
```

Comment:
Should a user be able to navigate to just the '/advisor/` page? Or should this redirect them to a specific tab?